VSRAD.Package/Commands/ActionsMenuCommand.cs

```diff
@@ -133,6 +144,11 @@ private async Task ExecuteActionAsync(ActionProfileOptions action)
                 var actionError = await _actionLogger.LogActionWithWarningsAsync(result).ConfigureAwait(false);
                 if (actionError is Error runError)
                     Errors.Show(runError);
+                _actionRuns = false;
+            }
+            catch (OperationCanceledException)
+            {
+                _actionRuns = false;
             }
             finally
             {
```

Comment:
I suggest you to move `_actionRuns = false;` to the `finally` block. This way, it is always executed when leaving the function and you don't need to repeat it multiple times.

Reply:
Thanks! Resolved by 1d60def.